packages/modules/pt/src/pages/employee/AssessmentDetails.js

```diff
@@ -68,7 +68,7 @@ const AssessmentDetails = () => {
       { Assessment: AssessmentData },
       {
         onError: (error, variables) => {
-          setShowToast({ key: "error", action: error });
+          setShowToast({ key: "error", action: error?.response?.data?.Errors[0]?.message || error });
```

Comment:
while creating assessment from employee ,  adhoc penalty and rebate option should be provided

Reply:
I will start working on it on new branch once Ajit confirm